.factory/automation.yml

```diff
@@ -171,7 +172,7 @@ release:
       image: typedb-ubuntu-22.04
       filter:
         owner: typedb
-        branch: [master, development]
+        branch: [master, development, "3.0"]
```

Comment:
Do we want to automatically sync dependencies from 3.0? I feel like it can break something still depending on 2.x typeql with default development branches in its repos